rust/azure_iot_operations_connector/src/destination_endpoint.rs

```diff
@@ -57,6 +65,11 @@ pub enum ErrorKind {
 pub(crate) struct Forwarder {
     message_schema_reference: Option<adr_models::MessageSchemaReference>,
     destination: ForwarderDestination,
+    device_uuid: Option<String>,
```

Comment:
No change really needed here, but consider `Option<'static &str>` in the future, especially for internal use. It makes certain things significantly smoother if there's no need for mutability.